pkg/template/api/zz_generated.deepcopy.go

```diff
@@ -84,6 +92,142 @@ func DeepCopy_api_Template(in interface{}, out interface{}, c *conversion.Cloner
+		}
+		if err := DeepCopy_api_TemplateBrokerUUIDMapSpec(&in.Spec, &out.Spec, c); err != nil {
+			return err
+		}
+		return nil
+	}
+}
+
+func DeepCopy_api_TemplateBrokerUUIDMapList(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*TemplateBrokerUUIDMapList)
+		out := out.(*TemplateBrokerUUIDMapList)
+		out.TypeMeta = in.TypeMeta
+		out.ListMeta = in.ListMeta
+		if in.Items != nil {
+			in, out := &in.Items, &out.Items
+			*out = make([]TemplateBrokerUUIDMap, len(*in))
+			for i := range *in {
+				if err := DeepCopy_api_TemplateBrokerUUIDMap(&(*in)[i], &(*out)[i], c); err != nil {
+					return err
+				}
+			}
+		} else {
+			out.Items = nil
+		}
+		return nil
+	}
+}
+
+func DeepCopy_api_TemplateBrokerUUIDMapSpec(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*TemplateBrokerUUIDMapSpec)
+		out := out.(*TemplateBrokerUUIDMapSpec)
+		out.TemplateInstanceRef = in.TemplateInstanceRef
+		if in.BindingIDs != nil {
+			in, out := &in.BindingIDs, &out.BindingIDs
+			*out = make([]string, len(*in))
+			copy(*out, *in)
+		} else {
+			out.BindingIDs = nil
+		}
+		return nil
+	}
+}
+
+func DeepCopy_api_TemplateInstance(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*TemplateInstance)
+		out := out.(*TemplateInstance)
+		out.TypeMeta = in.TypeMeta
+		if err := pkg_api.DeepCopy_api_ObjectMeta(&in.ObjectMeta, &out.ObjectMeta, c); err != nil {
+			return err
+		}
+		if err := DeepCopy_api_TemplateInstanceSpec(&in.Spec, &out.Spec, c); err != nil {
+			return err
+		}
+		if err := DeepCopy_api_TemplateInstanceStatus(&in.Status, &out.Status, c); err != nil {
+			return err
+		}
+		return nil
+	}
+}
+
+func DeepCopy_api_TemplateInstanceCondition(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*TemplateInstanceCondition)
+		out := out.(*TemplateInstanceCondition)
+		out.Type = in.Type
+		out.Status = in.Status
+		out.LastTransitionTime = in.LastTransitionTime.DeepCopy()
+		out.Reason = in.Reason
+		out.Message = in.Message
+		return nil
+	}
+}
+
+func DeepCopy_api_TemplateInstanceList(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*TemplateInstanceList)
+		out := out.(*TemplateInstanceList)
+		out.TypeMeta = in.TypeMeta
+		out.ListMeta = in.ListMeta
+		if in.Items != nil {
+			in, out := &in.Items, &out.Items
+			*out = make([]TemplateInstance, len(*in))
+			for i := range *in {
+				if err := DeepCopy_api_TemplateInstance(&(*in)[i], &(*out)[i], c); err != nil {
+					return err
+				}
+			}
+		} else {
+			out.Items = nil
+		}
+		return nil
+	}
+}
+
+func DeepCopy_api_TemplateInstanceSpec(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*TemplateInstanceSpec)
+		out := out.(*TemplateInstanceSpec)
+		if err := DeepCopy_api_Template(&in.Template, &out.Template, c); err != nil {
+			return err
+		}
+		out.SecretRef = in.SecretRef
+		out.ImpersonateUser = in.ImpersonateUser
```

Comment:
This should look like the struct in the certificates API upstream, and it would be "As" or "Requestor" or something similar.